.github/workflows/validate-pr.yml

```diff
@@ -28,10 +28,3 @@ jobs:
               run: npm run test:unit
             - name: Helm
               run: helm lint charts/substra-frontend
-
-    build: #separate for parallelization
```

Comment:
so we are no longer building for each PR, only on main?

Reply:
The new docker build workflow is called on PRs, so this bit here is now redundant.